modules/registry-exposing-secure-registry-manually.adoc

```diff
@@ -15,116 +15,44 @@ images using the route host.
 
 * The following prerequisites are automatically preformed:
 ** Deploy the registry.
-** Secure the registry.
 ** Deploy a router.
 
 .Procedure
 
-. Verify whether a passthrough route exists, the route should have been created by
-default for the registry during the initial cluster installation:
-+
-----
-$ oc get route/pod-registry -o yaml
-apiVersion: v1
-kind: Route
-metadata:
-  name: podman-registry
-spec:
-  host: <host> <1>
-  to:
-    kind: Service
-    name: podman-registry <2>
-  tls:
-    termination: passthrough <3>
-----
-<1> The host for your route. You must be able to resolve this name externally by
-DNS to the router's IP address.
-<2> The service name for your registry.
-<3> Specifies this route as a passthrough route.
-+
-[NOTE]
-====
-Re-encrypt routes are also supported for exposing the secure registry.
-====
+You can expose the registry insecurely or securely:
 
-.. If it does not exist, create the route with the `oc create route passthrough`
-command, specifying the registry as the route’s service. By default, the name of
-the created route is the same as the service name:
+. Using `DefaultRoute`, which is insecure:
```

Comment:
From testing - the default route is in fact secured, but is self-signed. Therefore in most use cases users logging into the registry will get an x509 certificate error.